server/sandbox.go

```diff
@@ -353,6 +354,13 @@ func (s *Server) RemovePodSandbox(ctx context.Context, req *pb.RemovePodSandboxR
 		s.removeContainer(c)
 	}
 
+	if err := label.UnreserveLabel(sb.processLabel); err != nil {
+		return nil, err
+	}
+	if err := label.UnreserveLabel(sb.mountLabel); err != nil {
```

Comment:
Only need the call for the processlabel.  SELinux is just grabbing the MCS portion and removing it, Calling this for mountlabel would have no effect.